source/layout/theme.liquid

```diff
@@ -10,6 +10,11 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
+  <script>
+    // remove no-js and add 'js' to the HTML
```

Comment:
allowances for styling for progressive enhancement and ensures that components styled for no-js situations won't flash on screen, before JS kicks in, and then the JS based styling takes over (as happens when people include this into their scripts at the bottom of files).

Again, biggest benefit is that this allows for styling hooks for people to render content if JS is disabled or blocked for some reason (think display: none content that is revealed via JS)